integration-tests/src/test/scala/tech/beshu/ror/integration/suites/FipsSslSuite.scala

```diff
@@ -57,12 +57,20 @@ class FipsSslSuite
 
   private lazy val rorClusterAdminStateManager = new CatManager(clients.last.adminClient, esVersion = esVersionUsed)
 
-  "Health check" should {
-    "be successful" when {
-      "internode ssl is enabled" in {
-        val response = rorClusterAdminStateManager.healthCheck()
+  // todo: The ES with FIPS does not start correctly when running tests on Windows with ES version lower than 8.18.
```

Comment:
It needs further investigation. My suspicion is that it is not an issue with tests, but with ROR itself. So I suspect Fips may not work on Windows for ROR with ES<=8.18. That is why I think fixing it is outside of scope of this task - it may not be test issue, but production code issue, that just never surfaced.